internal/git/git.go

```diff
@@ -32,10 +32,18 @@ var (
 	ErrUnknownGit         = errors.New("unknown git error")
 )
 
+type GitConfig struct {
+	User           string
+	Email          string
+	SigningKey     string
+	GPGImportPaths []string
```

Comment:
This field is not used in package `git` is it? Maybe we can move it out as a regular slice in `start.go` where it is used to load the keys?

Reply:
Yes, that makes sense. I will fix this